brig/cmd/brig/commands/run.go

```diff
@@ -86,9 +98,17 @@ var run = &cobra.Command{
 			return err
 		}
 
-		payload, err := readFileParam(runPayload)
-		if err != nil {
-			return err
+		if runPayload != "" && runJsonPayload != "" {
+			return errors.New("Both payload and json-payload should not be specified")
+		}
+
+		if runJsonPayload != "" {
+			payload := runJsonPayload
```

Comment:
This doesn't look right. Did this compile successfully? It should not have, because it appears that this `payload` variable goes out of scope before it is used.

Reply:
To clarify, a symbol named `payload` _is_ used down on line 135, but it's actually undefined because the `payload` variable you think you're referencing (i.e. the one here on line 106) is out of scope by then.

Reply:
Yup, fixed it!